cmd/kpod/create.go

```diff
@@ -230,6 +233,28 @@ func parseSecurityOpt(config *createConfig, securityOpts []string) error {
 		err       error
 	)
 
+	if config.pidMode.IsHost() {
+		labelOpts = append(labelOpts, label.DisableSecOpt()...)
+	}
+	if config.pidMode.IsContainer() {
```

Comment:
Can this be else-if? I think pid=host and pid=<container> are exclusive

Reply:
Fixed